src/components/result.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+
+const ResultView = ({ obj }) => (!obj ? (
+  <div className="result">
+    <p>0</p>
+  </div>
+) : (
+  <div className="result">
+
+    <p>
+      {obj.total}
+      {obj.operation}
+      {obj.next}
+    </p>
+  </div>
+));
+
+export default ResultView;
+
+ResultView.defaultProps = {
+  obj: {},
+};
+ResultView.propTypes = {
+  obj: PropTypes.shape({
+    total: PropTypes.string,
+    operation: PropTypes.string,
+    next: PropTypes.string,
+  }),
+};
```

Comment:
- [ ] When I make some calculations, the results that are long are cut off and the entire results don't show. For example, in the video below: I did 46/3 and got this is what showed:

![chrome-capture-2023-8-11](https://github.com/nilab-khwaja/math-magicians/assets/77106962/63d7ce6f-8d79-4560-88fb-81d50484cb2a)

Kindly fix this and ensure that the long calculations show correctly. 👍 😎